grails/start-grails

```diff
@@ -8,5 +8,5 @@ XMX=${JAVA_MAX_MEMORY-256}
 MAXPERM=${JAVA_MAX_PERM-128}
 
 cd /home/application/current
-grails prod run-app  --non-interactive --host=0.0.0.0 --port=${PORT}
+java -Dgrails.env=prod -jar build/libs/current-master.jar --server.host=0.0.0.0 --server.port=${PORT}
```

Comment:
is the output artifact name always current-master or does this depend on version?

There should be a property to define output name in the grails package command, that should do the trick instead of expecting master (although that most of the time this is true on tsuru).